app/api/server/v1/teams.ts

```diff
@@ -189,15 +189,24 @@ API.v1.addRoute('teams.listRoomsOfUser', { authRequired: true }, {
 API.v1.addRoute('teams.members', { authRequired: true }, {
 	get() {
 		const { offset, count } = this.getPaginationItems();
-		const { teamId, teamName } = this.queryParams;
-		const { query } = this.parseJsonQuery();
+		const { teamId, teamName, status, username, name } = this.queryParams;
+
+		if (status && !Array.isArray(status)) {
```

Comment:
You can use `check` here
`check(status, [String])`

Reply:
Great, much better! Thanks 🤗